### PR TITLE
🐛 Subscriptions: Stops stopping propagation of click events

### PR DESCRIPTION
**Pull request by ChrisAntaki**
Stopping the propagation of `click` events broke the `amp-bind` extension. This PR uses a new method to ensure `click` events are only handled once by the `amp-subscriptions` extension. Basically, an additional property, specific to `amp-subscriptions`, is added to the event when it is first handled, and the handler bails if it is present on a subsequent call.

I plan to test this PR across a variety of devices/browsers using the PR deploy tool described here:
https://github.com/ampproject/amp-github-apps/blob/master/pr-deploy/README.md

### CODE REVIEW

**Comment by ChrisAntaki** [comment]
This works with Chrome's mobile emulation. Now I'm testing it with real mobile devices

**Comment by jpettitt** [comment]
LGTM but please have @dvoytenko take a look as well to confirm the approach is ok within AMP as a whole

File: extensions/amp-subscriptions/0.1/local-subscription-platform-base.js [hunk 1]
```diff
@@ -109,27 +109,24 @@ export class LocalSubscriptionBasePlatform {
    * @protected
    */
   initializeListeners_() {
-    const handleClickAndStopEventPropagation = e => {
-      e.stopPropagation();
+    const handleClickOncePerEvent = e => {
+      if (e.ampSubscriptionsClickHasBeenHandled) {
```
Comment (dvoytenko) [inline extensions/amp-subscriptions/0.1/local-subscription-platform-base.js]: As a surrogate property we need to access it via `[]`. E.g. `e['_AMP_SUBS_CLICK_HANDLED']` or something like this. Would probably be best to put it in a const.

Comment (ChrisAntaki) [reply extensions/amp-subscriptions/0.1/local-subscription-platform-base.js]: Good call :+1:

**Pull request merged**